### DISCUSSION TITLE
Stepper motor vs Servo and slip ring

### DISCUSSION THREAD

**Discussion by mais01** [AlexAsplund/Vanchor#19] [General]
Hi Alex,

Interesting project, I am having a similar project ongoing. 
Currently i am only having manual operation, mainly due to mechanical challenges :) 
I also made the solution with a stepper motor, but i have utilized a belt, which have turned out to be head ache. (Too tight or too loose :)

Instead of utilizing back & forward rotation i thought to only use forward speed - as the propeller is much more efficient in that direction. I have used a slip-ring for the trolling-motor wires, which allows me to rotate freely the shaft unlimited amount of revolutions. 

I noted that you had also written in Issues that you are looking into a servo-option, I am on the same path to use a servo motor instead of a stepper motor. And obviously for me i would go for gears as well. But that would solve the position-trouble and also probably make it design wise better (due to smaller physical size).   

![SussiTrolling 1](https://user-images.githubusercontent.com/74763373/135034926-976253f8-fe6d-4d6e-8c33-03b173eb7588.jpg)
![SussiTrolling 2](https://user-images.githubusercontent.com/74763373/135034929-32807e8a-a9ff-49c4-8649-8a617df8f253.jpg)

**Comment by AlexAsplund** [comment]
Hi!

Great job and it looks really interesting! I found some mechanical challenges when designing the stepper version as well (the one in the repo is total makeover number 7).

The hardest part is getting the torque needed and fitting the gearbox that makes it possible.
Started out with a 8:1 gear at first, but it only worked if the propeller was turned of, a few versions later and I'm using a 36:1 gear ratio.

A slip-ring is a nice idea, I might use it in the servo design (or a big ball bearing), i found that the boat might move in unexpected ways when it reaches it's limit and turns almost 360 degrees in the opposite way😄

I just got an H bridge, AS5600 (magnetic rotary position sensor) and a 550 motor that will come in handy for building and designing the servo :)

**Comment by AlexAsplund** [comment]
Soon ready with the design of this. Just got to redesign some gears that magically went missing.
No slip ring in this one though, they are hard to get around here and ordering them from china is pretty expensive, but it's pretty easy to add in afterwards.

![image](https://user-images.githubusercontent.com/12714125/135625175-0b46e9a7-6572-4e3c-9c4f-f65b6bca4a12.png)
